### DISCUSSION TITLE
A collection of proposed new features

### DISCUSSION THREAD

**Discussion by pastelsky** [pastelsky/bundlephobia#447] [Ideas]
These are feature requests that have been raised by various users. However, I cannot work on all of them by myself, hence collecting them here to gauge community interest in it being built / building it.

### Support for remote (non-npm) packages

1. Support for github repositories / part of repositories as packages (https://github.com/pastelsky/bundlephobia/issues/65)

**Comment by enessoylu** [comment]
It could be really cool to compare two or more packages, for example I want to use a debounce hook for React and I can use react-use or @uidotdev/usehooks and I want to consider the sizes and such to decide.
